src/axom/primal/operators/compute_moments.hpp

```diff
@@ -10,7 +10,7 @@
  * \file compute_moments.hpp
  *
  * \brief Consists of a set of methods to compute areas/volumes and centroids 
- * for Polygons and CurvedPolygons composed of BezierCurves
+ * for Polygons and CurvedPolygons composed of nonrational BezierCurves
```

Comment:
```suggestion
 * for Polygons and Curved Polygons composed of non-rational Bezier Curves
```

Reply:
I think @jcs15c was keying off of a paper that he used for reference when implementing some of these routines, which uses `nonrational` vs. `rational` (see below). I've also seen "polynomial Bezier curves" vs. "rational Bezier curves" when distinguishing between the two.
I think we should avoid `irrational` since that implies "cannot be expressed as a ratio" (e.g. `sqrt(2)`) rather than "is not currently expressed as a ratio".  

See: https://github.com/LLNL/axom/blob/8129e1ea791aa4de51e9fde52941ebfed53bd1d5/src/axom/primal/geometry/BezierCurve.hpp#L53-L57

Reply:
I'm not sure why, but it seems like the convention is to use the phrase "Nonrational Bezier Curves" over "Irrational" or even "Polynomial Bezier Curves." Or at least, the both the Farin 83 reference reference I cite above and the NURBS book both prefer the phrase "nonrational."

Reply:
OK. It seems a bit odd to me, but if it's common usage.